fb_apt/providers/sources_list.rb

```diff
@@ -0,0 +1,70 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+def whyrun_supported?
+  true
+end
+
+use_inline_resources
+
+action :run do
+  repos = []
+  mirror = node['fb_apt']['mirror']
+  distro = node['lsb']['codename']
+
+  # only add base repos if mirror is set and codename is available
+  if mirror && distro
+    components = 'main'
+    if node.ubuntu?
+      components += ' universe'
```

Comment:
So my understanding is that universe is like main, but the packages are not necessarily officially supported by ubuntu. Every single ubuntu system I've seen in the wild has it enabled, and the license is the same as main, so IMO it makes sense to enable it here.